pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -45,6 +48,8 @@ const (
 	maxRetryDelay = 300 * time.Second
 	// Default number of workers processing a node change
 	defaultWorkers = 4
+
+	ovsExternalIDNodeName = "node-name"
```

Comment:
Now ovs external ID constants are spread in files, so do ParseXXXInterfaceConfig. Do you think we could put them together and closer to interfaceCache?

Reply:
Then why you suggest to put them (and buildXXXExternalIDs)? I do not think externalIDs belong to InterfaceStore, which is just a data store for interface data, and should not know anything about OVS.

Reply:
No strong opinion, just felt a place having all external ID constants and the parsing functions might be convenient to know what data antrea persist in ovsdb and save class like AgentInitializer to depend on all packages that have parsing function. I didn't mean putting them in interfaceCache, but a separate package or file closer to interfaceCache as they are the things stored and read in interfaceCache.

Reply:
I personally do not like to create too many small pkgs, but do not have a good idea which existing pkg for them.

Reply:
I was considering a separate file in `pkg/agent/interfacestore`. But I'm not sure if it makes sense in other perspectives like the interface scope you mentioned. I'm fine with keeping them as they are until we are clear that the other way were more appropriate.